webui/react/package.json

```diff
@@ -33,9 +33,9 @@
     "portable-fetch": "^3.0.0",
```

Comment:
We should consider updating the other deps too?

<img width="629" alt="Schermata 2021-04-27 alle 09 11 00" src="https://user-images.githubusercontent.com/5413702/116200524-bb75d180-a738-11eb-868b-b12f02b2234f.png">

Reply:
I have a separate ticket / story for the task of upgrading the rest just in case it became a huge ordeal. But want to tackle this soon too.